Containers/mastercontainer/start.sh

```diff
@@ -70,6 +70,7 @@ fi
 # Check if api version is supported
 if ! sudo -u www-data docker info &>/dev/null; then
     print_red "Cannot connect to the docker socket. Cannot proceed."
+    echo "Did you maybe remove group read permissions for the docker socket? AIO needs them in order to access the docker socket."
```

Comment:
> Did you maybe remove group read permissions

This should read:

> Did you maybe remove group write permissions

Reply:
I think read permissions should be enough.

Reply:
@szaimen ... did you test it? because I did, and it was the write permission that it needed...